packages/catalog-generator/scripts/build-catalogs.mjs

```diff
@@ -13,9 +13,9 @@ const require = createRequire(import.meta.url);
  * @type {Record<import('../dist/types').CatalogRuntime, string[]>}
  **/
 const CATALOGS = {
-  Main: ['4.8.1', '4.4.4', '4.4.0.redhat-00046', '4.8.0.redhat-00017'],
+  Main: ['4.8.2', '4.4.4', '4.4.0.redhat-00046', '4.8.0.redhat-00017'],
   Quarkus: ['3.16.0', '3.8.3', '3.8.0.redhat-00018'],
-  SpringBoot: ['4.8.1', '4.4.4', '4.4.0.redhat-00039', '4.8.0.redhat-00022'],
+  SpringBoot: ['4.8.2', '4.4.4', '4.4.0.redhat-00039', '4.8.0.redhat-00022'],
 };
 const KAMELETS_VERSION = '4.8.1';
```

Comment:
I am curious why Kamelets are not having a 4.8.2. Do you know?

Reply:
it is a separated release from the Camel Core (like Camel Quarkus) I guess it is a matter of time to put it under vote.
For our case, we will move to 4.9.0 anyway